packages/select/src/__examples__/films.tsx

```diff
@@ -138,12 +138,13 @@ export const TOP_100_FILMS: Film[] = [
  */
 export function getFilmItemProps(
     film: Film,
-    { handleClick, handleFocus, modifiers, query, ref }: ItemRendererProps,
-): MenuItemProps & React.Attributes {
+    itemProps: ItemRendererProps,
+): Omit<MenuItemProps, "key"> & React.Attributes {
```

Comment:
If we're omitting `key` from `MenuItemProps` for the return type, we should also remove `React.Attributes` since the intersection of that type just ends up adding it back:
```ts
interface Attributes {
    key?: Key | null | undefined;
}
```
```suggestion
): Omit<MenuItemProps, "key"> {
```

I'm supportive of this fix for the docs examples, but could we split that part out into its own PR?

Reply:
That makes sense, thank you. I mostly am keeping in the docs examples as it is useful for my testing. Unless I'm not supposed to test on the docs examples? What is the better way to test?